### PR TITLE
Add my first file to learn the formatting

### PR DESCRIPTION
**Pull request by superojla**
**Description**

First document to learn the formatting in Markdown.

**Related issue(s)**
#1

### CODE REVIEW

File: JustynaTest.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+### Oraz podsekcja
+
+A tak sobie sklonowałam repozytorium:
+
+```bash
+git clone {REPO_URL}
+```
+
+Powrót do [pierwszej sekcji](#to-jest-tytul-sekcji)
```
Comment (superojla) [inline JustynaTest.md:52]: Nie działa, naprawić.

File: JustynaTest.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
+Autor: 
+    Imię: Justyna
+    github name: superojla
+Data: 15.09.2020
+---
+#To jest tytuł
```
Comment (majakurcius) [inline JustynaTest.md:7]:
```suggestion
# To jest tytuł
```

File: JustynaTest.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  <summary label="LABELKA2">
+  Tytuł 2
+  </summary>
+  A Szczebrzeszyn z tego słynie.
+  </details>
+  <details>
+  <summary label="LABELKA3">
+  Tytuł 3
+  </summary>
+  Nie pamiętam, jak to dalej leci. [wierszyk o chrząszczu w Szczebrzeszynie](https://www.tenpieknyswiat.pl/2007/05/09/w-szczebrzeszynie-chrzaszcz-brzmi-w-trzcinie)
```
Comment (majakurcius) [inline JustynaTest.md:36]:
```suggestion
  Nie pamiętam, jak to dalej leci. [Wierszyk o chrząszczu w Szczebrzeszynie](https://www.tenpieknyswiat.pl/2007/05/09/w-szczebrzeszynie-chrzaszcz-brzmi-w-trzcinie)
```

File: JustynaTest.md (hunk 1 above)
Comment (majakurcius) [inline JustynaTest.md:52]:
```suggestion
Powrót do [pierwszej sekcji](#to-jest-tytul-sekcji).
```

File: JustynaTest.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  </details>
+</div>
+
+## Nowa sekcja
+
+Tu nie powinno być pusto!
```
Comment (majakurcius) [inline JustynaTest.md:42]: Zgadza się!

**Review by majakurcius — COMMENTED** [review]
Test review

File: JustynaTest.md (hunk 4 above)
Comment (superojla) [reply JustynaTest.md:42]: Super!

Comment (superojla) [reply JustynaTest.md:42]: /meow

File: JustynaTest.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+```bash
+git clone {REPO_URL}
+```
+
+Powrót do [pierwszej sekcji](#to-jest-tytul-sekcji).
+
+## Listy
+
+### Lista nieuporządkowana
```
Comment (superojla) [inline JustynaTest.md:56]:
```suggestion
### Lista punktowa
```

File: JustynaTest.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+- Chleb
+- Słodycze
+    - Czekolada
+    - Szkloki
+        - Winogronowe
+
+* Kot
+* Pies 
+* Kuń
+* Koza
```
Comment (superojla) [inline JustynaTest.md:69]:
```suggestion
* Koza
* Kuń
* Pies 
* Kot
```

File: JustynaTest.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+  <summary label="LABELKA2">
+  Tytuł 2
+  </summary>
+  A Szczebrzeszyn z tego słynie.
+  </details>
+  <details>
+  <summary label="LABELKA3">
+  Tytuł 3
+  </summary>
+  Nie pamiętam, jak to dalej leci.
```
Comment (superojla) [inline JustynaTest.md:36]:
```suggestion
  Nie pamiętam, jak to dalej leci. <a href="https://www.tenpieknyswiat.pl/2007/05/09/w-szczebrzeszynie-chrzaszcz-brzmi-w-trzcinie">Wierszyk o chrząszczu w Szczebrzeszynie</a>
```

**Pull request merged**